intermittent_failures.md

```diff
@@ -0,0 +1,95 @@
+
+If you are blocked, you own fixing what blocks you and/or finding support. Even if you are not blocked, the community will appreciate fixes or feedback submitted to the appropriate SIG. Which SIG owns a failure is not always obvious, as the Automated Review pipeline executes code maintained by every SIG in O3DE. If you have questions, reach out over [Discord](https://discord.gg/p3padwr58u)!
+
+## Policy
+
+With few exceptions, no intermittently failing test or feature is allowed. If you see a failure that appears nondeterministic, immediately remove it following these steps:
+
+1. Determine if the failure is intermittent
+   - If possible, execute on your local machine multiple times to tally a failure rate, and attempt to debug the issue
+   - Execute the Jenkins pipeline as many as 10 times, and save at least one text file with a failing log
+2. The source of intermittent failure should be immediately removed from the pipeline, and tagged unstable
+   - If possible, create a new pull request which reverts or disables the intermittent module, feature, test, or hardware-configuration
+3. [Cut a GitHub Issue](https://github.com/o3de/o3de/issues/new/choose) to track both fixing the underlying issue and re-enabling the test
+   - Add a label for the SIG which maintains that feature 'sig/XYZ' ***OR*** use label 'needs-sig'
+
+### Exceptions
+
+The policy above applies to all code, scripts, and hardware used in the Automated Review Pipeline with the following exceptions:
+
+- It is rarely more risky to remove an unstable critical test (or feature) than to leave it in a known intermittently failing state
+  - If you have a reason to not follow this policy, please reach out to SIG-Testing on [Discord](https://discord.gg/p3padwr58u)!
+- Intermittent failures ocurring on only one Operating System are still intermittent failures, though it is reasonable to conditionally disable
+- Builds or tests which reliably fail at a given commit are **not** considered intermittent
+  - Such failures may appear inconsistent across multiple commits, when thrashed by rapid code changes
+  - While the pipeline makes these difficult to introduce, when they occur it is still appropriate to immediately revert or disable until fixed
+
+## The Sandbox Suite
```

Comment:
Does the Sandbox suite apply to only the Python test cases or does it apply to GoogeTest test cases as well? If it applies only to python test cases, If would also be good to move this information to Step 4 below by renaming Step 4 from "Disable" to "Sandbox or Disable" and document how to Sandbox python test cases and disable google test test cases.

Reply:
Sandbox should be used when a fix is being tested, and needs more information. I think it is appropriate to first disable, and then later move into Sandbox when a fix has been applied but needs more information on its stability. I can make that recommendation more clear.